### PR TITLE
bump dka 1.2.1 (#350)

### PR DESCRIPTION
**Pull request by joejulian**
* chore: adding revision

* chore: bump dka

Co-authored-by: Joe Julian <me@joejulian.name>
(cherry picked from commit 4125bb52541cd1f3a50f6b9aa8086440ea573dfc)

**What type of PR is this?**

backport

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/dex-k8s-authenticator/1.2.x/dex-k8s-authenticator-2.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: dex-k8s-authenticator
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: dex-k8s-authenticator
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.1-1"
```
Comment (alejandroEsc) [inline addons/dex-k8s-authenticator/1.2.x/dex-k8s-authenticator-2.yaml:10]: revision needs to be bumped

Comment (joejulian) [reply addons/dex-k8s-authenticator/1.2.x/dex-k8s-authenticator-2.yaml:10]: It was - in master. This PR just cherry-picks the squashed commit.

Comment (joejulian) [reply addons/dex-k8s-authenticator/1.2.x/dex-k8s-authenticator-2.yaml:10]: Went from 1.2.0-1 (in dex-k8s-authenticator-1.yaml) to 1.2.1-1

Comment (alejandroEsc) [reply addons/dex-k8s-authenticator/1.2.x/dex-k8s-authenticator-2.yaml:10]: ok.

**Pull request merged**